src/scripts/webview.js

```diff
@@ -41,6 +42,24 @@ class WebView extends EventEmitter {
                 }, '*')
             `);
         });
+
+        Mousetrap.bind(['command+left', 'ctrl+left'], () => {
+            const webviewObj = this.getActive();
+            webviewObj.executeJavaScript(`
+                window.history.back();
```

Comment:
Does the [built in electron method](https://electron.atom.io/docs/api/webview-tag/#webviewgoback) not work here?